portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,134 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Erik Rodriguez</title>
+	<link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+
+	<div class="navigation-container">
+		<header>
+			<a href="index.html" class="main-logo"> Erik Rodriguez </a>
+			<nav>
+				<ul>
+					<li> <a href="#"> Work </a> </li> <!---TODO: ADD scroll-to functionality to each link-->
+					<li> <a href="#"> Skills </a> </li>
+					<li> <a href="#"> Contact Me </a> </li>
+				</ul>
+			</nav>
+		</header>
+		<div class="social-header">
+			<ul>
+				<li><a href="#" > <img src="images/github.png" alt="github"></a></li>
+				<li><a href="#" > <img src="images/linkedin.png" alt="linkedin"></a></li>
+				<li><a href="#" > <img src="images/instagram.png" alt="instagram"></a></li>
+			</ul>
+		</div>
+	</div>
+
+	<div class="hero">
+		<div class="content">
+            <h1 class="stagger1">From Great Ideas To Beautiful Experiences</h1>
+            <div class="meet stagger1">
+                <span>👇</span>
+                <p>Meet Erik Rodriguez</p>
+            </div>
+            <svg class="scroll stagger1" width="40" height="77" viewBox="0 0 40 77">
+                <g id="scroll" transform="translate(-253 -787)">
+                  <g id="Rectangle_12" data-name="Rectangle 12" transform="translate(253 787)" fill="none" stroke="#fff" stroke-width="4">
+                    <rect width="40" height="77" rx="20" stroke="none"/>
+                    <rect x="2" y="2" width="36" height="73" rx="18" fill="none"/>
+                  </g>
+                  <circle class="circle" id="Ellipse_1" data-name="Ellipse 1" cx="11" cy="11" r="11" transform="translate(262 798)" fill="#fff"/>
+                </g>
+              </svg>
+		</div>
+        <svg xmlns="http://www.w3.org/2000/svg" class="hero-design" width="686" height="688" viewBox="0 0 686 688">
+            <g id="blockdesign" transform="translate(-935 -289)">
+              <rect class="square-anim" data-name="Rectangle 2" width="172" height="172" rx="19" transform="translate(1277 289)" fill="#1616AC"/>
```

Comment:
It is recommended to prefer lower case for color codes and throughout, see https://google.github.io/styleguide/htmlcssguide.html#Capitalization